src/OpenFeature/Telemetry/EvaluationEventBuilder.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using OpenFeature.Constant;
+using OpenFeature.Model;
+
+namespace OpenFeature.Telemetry;
+
+/// <summary>
+/// Class for creating evaluation events for feature flags.
+/// </summary>
+public static class EvaluationEventBuilder
```

Comment:
Consider converting `EvaluationEventBuilder` to a non-static class and introducing a static `Default` instance:

```csharp
public sealed class EvaluationEventBuilder
{
    public static EvaluationEventBuilder Default { get; } = new();

    public EvaluationEvent Build(HookContext<Value> hookContext, FlagEvaluationDetails<Value> details)
    {
        ...
    }
}
```

This change would allow us to inject different implementations (e.g., enriched or test-specific builders) if needed, and enables cleaner mocking or extension scenarios in the future.

Reply:
Fixed with https://github.com/open-feature/dotnet-sdk/pull/397/commits/b7d34fb01e5bd3149139960aea3a19a8ae0a3b6b